templates/overrides/menu/menu.html.twig

```diff
@@ -4,5 +4,6 @@
  */
 #}
 {{ pattern('navigation', {
-  'items': items
+  'variant': 'pills',
```

Comment:
why do we use pills ?

Reply:
At netlify the default navigation has pills: https://oelibrary.netlify.app/default/?path=/story/components-navigation--default&args=alignment:!undefined

Reply:
Well, since there is not specific requirement we can go with that